cli/cli_test.go

```diff
@@ -44,12 +44,12 @@ func TestCWRCommands(t *testing.T) {
 	c := newTestCLI(t)
 
 	// check 'meta cwr convert' prints a CID
-	stdout := c.run("cwr", "convert", "../cwr/testdata/testfile.cwr", "../cwr/CWR-DataApi")
-	id, err := cid.Parse(strings.TrimSpace(stdout))
+	stdout := c.run("cwr", "convert", "../cwr/testdata/testfile.cwr")
+	id, err := cid.Parse(strings.Split(stdout, "\n")[0])
```

Comment:
Why has this changed? The test needs to make sure *only* the CID is printed (optionally surrounded by whitespace), otherwise it breaks using the output as a stream.

Reply:
When adding another record to be index ...the test produce 2 CIDs.
cid.Parse fail without this change

Reply:
Ok, so test that exactly two expected CIDs are on stdout then?

Reply:
we can do that ...though each time we will add a field the test will be changed.

Reply:
ok, so the choices are to either keep updating this test, or convert a file which doesn't change. Whatever we do, this test should ensure that the output of the program is _exactly_ what we expect.